themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig

```diff
@@ -0,0 +1,95 @@
+ *     full: "node--view-mode-full".
+ *   The following are controlled through the node publishing options.
+ *   - node--promoted: Appears on nodes promoted to the front page.
+ *   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+#}
+<a href="{{ url }}" class="group block flex flex-row-reverse w-full border-b-0 bg-white text-grey-800 drop-shadow-xl text-left rounded-xl overflow-hidden transition-shadow hover:shadow-halo">
+  {% if content.field_teaser_image|render or content.field_image|render %}
+    <div class="hidden md:block w-24 grow-0 shrink-0 relative">
+      <div class="h-full child:!h-full child:object-cover">
+        {% if content.field_teaser_image|render %}
+          {{ content.field_teaser_image|field_value }}
+        {% else %}
+          {{ content.field_image|field_value }}
+        {% endif %}
+      </div>
+      <div class="absolute inset-0 flex items-center justify-center opacity-0 group-hover:opacity-100 group-hover:backdrop-blur-[2px] transition-all bg-primary/50">
+        <span class="inline-block text-4xl text-primary-contrast">&#8594;</span>
+      </div>
+    </div>
+  {% endif %}
+
+  {% set title = node.label %}
+  {% if content.field_teaser_title|render %}
+    {% set title = node.field_teaser_title.0.value|check_markup('full_html') %}
+  {% endif %}
+
+  <div class="px-4 sm:px-6 py-3 flex items-center grow">
+    <p>
+      <span class="block text-xs text-grey-500 leading-relaxed">{{ 'Up next:'|t }}</span>
+      <span class="block text-sm md:text-base leading-snug font-medium">{{ title }}</span>
+    </p>
+  </div>
+</a>
```

Comment:
We should try and avoid using `<a>` tags to wrap big blocks of content - when a screen reader reaches this teaser it's going to read out information about the decorative image image as well as the 'up next: _title_' part, which is quite a lot of information to digest. 

As alternative, we could make only the 'up next: _title_' text (or perhaps even just the title text) a link and then use psudeo element on the link that is positioned over the entire teaser, which would give the whole teaser a clickable / focusable state whilst keeping the link text concise. And example of this approach can be found on the teasers on the BBC news homepage

Reply:
I like this idea and was easy to implement.